lib/debounce/index.js

```diff
@@ -23,11 +23,7 @@ export default function debounce(time, fn, opts) {
       }
     } else {
       clearTimeout(timer)
-      if (opts.leading) {
-        clear()
-      } else {
-        call(arguments)
-      }
+      call(arguments)
```

Comment:
maybe add JSDoc here for clear understanding of what `leading` is for?
and options can be described as well